lib/matterStacks.ts

```diff
@@ -89,16 +89,20 @@ export class MatterStack extends Stack {
             // Create PAI
             let prodIds = new CfnParameter(this, "productIds", {
                 type: "List<String>",
-                description: "A list of product IDs to create first PAIs"
+                description: "A list of product IDs to create first PAIs. These must be 4-digit hex values."
             })?.valueAsList;
             const vendorId = new CfnParameter(this, "vendorId", {
                 type: "String",
-                description: "If paaArn parameter is not provided, then this vendorId is used to create PAI(s)"
+                description: "If paaArn parameter is not provided, then this vendorId is used to create PAI(s). This must be a 4-digit hex value."
```

Comment:
Interesting, paaArn actually cannot be not provided when creating PAIs..

Sorry, haven't noticed it before.